LESSONS.md

```diff
@@ -0,0 +1,7 @@
+2020-08-18 08:49:56
+
+# Lessons Learned
```

Comment:
Do we have any other lessons we want to include here?

Reply:
Oofda... yeah probably 😆 . I can't think of any other big ones at the moment, but we can be an effort to try and add things to this as we come across nuances and such.